<!DOCTYPE html>
<html lang="en-us">
<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=edge">
    <title>Sohaib/research/wqfmtree/</title>
    
    <meta name="viewport" content="width=device-width, initial-scale=1">
    <meta name="robots" content="all,follow">
    <meta name="googlebot" content="index,follow,snippet,archive">
    <link rel="stylesheet" href="https://sohaib03.github.io/hugo-theme-console/css/terminal-0.7.4.min.css">
    <link rel="stylesheet" href="https://sohaib03.github.io/hugo-theme-console/css/animate-4.1.1.min.css">
    <link rel="stylesheet" href="https://sohaib03.github.io/hugo-theme-console/css/console.css">
    
      <!--[if lt IE 9]>
          <script src="https://oss.maxcdn.com/html5shiv/3.7.2/html5shiv.min.js"></script>
          <script src="https://oss.maxcdn.com/respond/1.4.2/respond.min.js"></script>
      <![endif]-->
       <meta property="og:title" content="WQFM-tree" />
<meta property="og:description" content="" />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://sohaib03.github.io/research/wqfmtree/" /><meta property="article:published_time" content="2024-07-31T00:00:00+00:00" />



<meta name="twitter:title" content="WQFM-tree"/>
<meta name="twitter:description" content="Overview
wQFM-TREE is a summary method developed for species tree estimation from multi-locus data, addressing gene tree discordance. It builds on the wQFM method and improves its scalability, making it applicable to large datasets."/>

</head>
<body class="terminal">
    <div class="container">
        <div class="terminal-nav">
          <header class="terminal-logo">
            <div class="logo terminal-prompt">
              
              
              <a href="https://sohaib03.github.io/" class="no-style site-name">Sohaib</a>:~# 
              <a href='https://sohaib03.github.io/research'>research</a>/<a href='https://sohaib03.github.io/research/wqfmtree'>wqfmtree</a>/</div></header>
          <nav class="terminal-menu">
            <ul vocab="https://schema.org/" typeof="BreadcrumbList">
                
                <li><a href="https://sohaib03.github.io/about/" typeof="ListItem">about/</a></li>
                
                <li><a href="https://sohaib03.github.io/research/" typeof="ListItem">research/</a></li>
                
                <li><a href="https://sohaib03.github.io/projects/" typeof="ListItem">projects/</a></li>
                
            </ul>
          </nav>
        </div>
    </div>

    <div class="container animated zoomIn fast" >
        
<h1>WQFM-tree</h1>

Jul. 31, 2024


<br/><br/>
<h3 id="overview">Overview</h3>
<p><strong>wQFM-TREE</strong> is a summary method developed for species tree estimation from multi-locus data, addressing gene tree discordance. It builds on the wQFM method and improves its scalability, making it applicable to large datasets.</p>
<h3 id="key-features">Key Features</h3>
<ul>
<li><strong>Improved Scalability:</strong> Removes the need for explicit quartet generation and weighting, allowing efficient processing of large datasets with many taxa and genes.</li>
<li><strong>Polytomy Handling:</strong> Unlike wQFM, wQFM-TREE can handle datasets with polytomies.</li>
<li><strong>Accuracy:</strong> In simulations, wQFM-TREE matches or outperforms ASTRAL in terms of accuracy across various model conditions.</li>
<li><strong>Real-World Application:</strong> Successfully re-analyzed the green plant dataset from the One Thousand Plant Transcriptomes Initiative, demonstrating its practical utility.</li>
<li><strong>Algorithmic Advances:</strong> Introduces improvements in computational methods, benefiting various quartet-based computations in phylogenetic analyses.</li>
</ul>
<h3 id="performance">Performance</h3>
<p>Simulations show that wQFM-TREE outperforms ASTRAL in 25 out of 27 conditions tested, with significant improvements in 20 of these conditions. It demonstrates reliability and accuracy, even in challenging phylogenetic models involving 200-1000 taxa.</p>
<h3 id="applications">Applications</h3>
<p>wQFM-TREE offers an effective alternative to existing methods, with proven scalability and accuracy in large datasets, advancing phylogenetic estimation.</p>
<h3 id="availability">Availability</h3>
<p>wQFM-TREE is available as open-source software.<br>
<a href="https://github.com/navidh86/wQFM-TREE">GitHub Repository</a></p>
<h3 id="citation">Citation</h3>
<p>Please cite the following publication if you use wQFM-TREE in your work. Additional details can be found in the preprint on <strong>bioRxiv</strong>:<br>
<a href="https://www.biorxiv.org/content/10.1101/2024.11.11.622962v2">Read on bioRxiv</a></p>


        <div class="footer">
    Powered by <a href="https://gohugo.io/">Hugo</a> with
    <a href="https://github.com/mrmierzejewski/hugo-theme-console/">Console Theme</a>. 
</div>

    </div>
  </body>
</html>
